src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure implements Properties {
```

Comment:
if you implement this interface in Figure class you don't have to implement it in each separate figure class because the will inherit it